lib/Config/Populator.php

```diff
@@ -276,16 +273,29 @@ private function createDefaultUsersFlags() {
     if (!empty($users_seen_editor_tutorial)) {
       foreach ($users_seen_editor_tutorial as $setting) {
         $user_id = substr($setting->name, $prefix_length);
-        UserFlag::createOrUpdate([
-          'user_id' => $user_id,
-          'name' => 'editor_tutorial_seen',
-          'value' => $setting->value,
-        ]);
+        $this->createOrUpdateUserFlag($user_id, 'editor_tutorial_seen', $setting->value);
       }
       Setting::whereLike('name', $prefix . '%')->deleteMany();
     }
   }
 
+  private function createOrUpdateUserFlag($user_id, $name, $value) {
+    $user_flags_repository = \MailPoet\DI\ContainerWrapper::getInstance(WP_DEBUG)->get(UserFlagsRepository::class);
```

Comment:
That was causing serialization error in tests. But that was before I made connection serializable so I think I can now. I'll try, good catch.

Reply:
Actually, I'd prefer not doing it now since the PR is already big and this would require changes in Premium (changing Populator constructor) and a bunch of tests.